plinth/views.py

```diff
@@ -30,8 +30,28 @@
 
 from . import forms, frontpage
 import plinth
+from django.utils.translation import ugettext_lazy as _
 
+from plinth import cfg
 
+def apps_init():
```

Comment:
apps_init and sys_init should be in ```plinth/__main__.py```.

But also, I don't think we need separate functions for these. We can just move the cfg.main_menu.add_urlname() calls into main(), maybe right before the modules are loaded.